howso/utilities/utilities.py

```diff
@@ -39,7 +39,7 @@
 CaseIndices = Iterable[Union[List[Union[str, int]], Tuple[Union[str, int]]]]
 DATETIME_TIMEZONE_PATTERN = re.compile(r"(?<!%)(?:%%)*(%z)", re.IGNORECASE)
 DATETIME_UTC_Z_PATTERN = re.compile(r"\dZ$")
-EPOCH = dt.datetime.utcfromtimestamp(0)
+EPOCH = dt.datetime.fromtimestamp(0, dt.UTC)
```

Comment:
datetime.UTC was added in python 3.11, should use datetime.timezone.utc instead.